main.go

```diff
@@ -156,6 +156,7 @@ func mainWithError() error {
 	daemonCommand.PersistentFlags().String(f.Service.Resource.ConfigMap.Key, "prometheus.yml", "Key in configmap under which prometheus configuration is held.")
 	daemonCommand.PersistentFlags().String(f.Service.Resource.ConfigMap.Name, "prometheus", "Name of prometheus configmap to control.")
 	daemonCommand.PersistentFlags().String(f.Service.Resource.ConfigMap.Namespace, "monitoring", "Namespace of prometheus configmap to control.")
+	daemonCommand.PersistentFlags().Duration(f.Service.Resource.ConfigMap.ReloadWaitTime, 10*time.Second, "Time to wait before reloading prometheus, to allow for configmap propagation.")
```

Comment:
wow I hope we don't have to use 10 seconds, that seems pretty long.

Reply:
We are secure :D